ci/utils/test.cue

```diff
@@ -33,6 +33,29 @@ import (
 	}
 }
 
+#TestImageCommand: {
```

Comment:
What were you planning to use this for?

Reply:
No, no, that's cool - it's just that the command isn't actually being invoked, I think 😄

Reply:
Want to pair on this tomorrow and get it over the line? Might have to test things a bit and can make updates in real-time.

Reply:
Paired and resolved